### ISSUE TITLE
No longer working

### ISSUE THREAD

**Issue by frishtik** [farukalampro/yelp-webscraper-using-scrapy-python-spider#1]
Great idea for a project.
However, I did follow all instructions & it does run perfectly fine, but just creates an empty CSV...
Could it be an issue with website architecture changes? Your project was published not so long ago so it's odd.

**Comment by farukalamai** [comment]
yes, website architecture is changed. Try to learn about how to select class, id to work with web scraping.

**Comment by frishtik** [comment]
Got it! Thanks for the quick reply!
One more thing: how do I set rotating proxies? Yelp seems to immediately block any scraping tries... 
I see that the settings.py is quite different from the ordinary scrapy settings, is that right? I tried to run [scrapy-rotating-proxies](https://github.com/TeamHG-Memex/scrapy-rotating-proxies) with a proxy list I found online but it doesn't seem to work...
Did you happen to play with this stuff yourself? I guess anyone who scrapes Yelp would face the same issue.
Thanks anyway for this!
